modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -264,4 +259,145 @@
             width: var(--e-contact-buttons-size-large);
         }
 	}
+
+    &__message-bubble {
+        background-color: var(--e-contact-buttons-message-bubble-chat-bg);
+        padding: 20px 20px 20px 40px;
+    }
+
+    &__bubble {
+        background-color: var(--e-contact-buttons-message-bubble-bubble-bg);
+        border-radius: 15px;
+        padding: 20px;
+        position: relative;
+
+        &::after {
+            border-color: transparent var(--e-contact-buttons-message-bubble-bubble-bg) transparent transparent;
+            border-style: solid;
+            border-width: 0 40px 40px 0;
+            content: "";
+            height: 0;
+            position: absolute;
+            left: -20px;
```

Comment:
`left: -20px;`
RTL?

Reply:
fixed. I have used the check `@if $direction == rtl`. Please let me know if there's an alternative approach that you usually use for rtl.

Reply:
@mserino 

`#{$start}: -20px;`